client/cli/src/params/network_params.rs

```diff
@@ -125,6 +130,12 @@ impl NetworkParams {
 			self.listen_addr.clone()
 		};
 
+		let public_addresses = if self.public_addr.is_empty() {
+			Vec::new()
+		} else {
+			self.public_addr.clone()
+		};
```

Comment:
```suggestion
		let public_addresses = self.public_addr.clone();
```

I think you can just write `self.public_addr.clone()` because if it's empty you will get an empty vec and if it's not you will get well... a filled vec. Or am I missing something? :confounded:

Reply:
Yeah, good call! Thanks.